frontend/ui/lib/widgets/trip_list_widget.dart

```diff
@@ -27,9 +27,18 @@ class TripListWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ListView(
-      children: trips.map(toListTile).toList(),
-    );
+    return Column(children: [
+      RaisedButton(
+        onPressed: () {
+          Navigator.pushNamed(context, '/trips/new');
+        },
+        child: Icon(Icons.add),
+      ),
+      Expanded(
+          child: ListView(
+        children: trips.map(toListTile).toList(),
```

Comment:
the spacing looks odd to me here - did the formatter do this?

Reply:
Yeah the formatter is really weird to me because it occasionally reformats something it already formatted or a file that isn't even being worked on but I'll run again.